apps/styleguide/src/layout/Panel.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div
+    class="bg-white p-2 border border-light rounded-sm"
+    :class="{'m-n2': outside}"
+  >
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    outside: {
```

Comment:
would outsidePadding not be a better name for this?

Reply:
agreed